src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -149,8 +161,15 @@ abstract class IntegrationTestBase {
     hmppsQueueService.findByQueueId("educationandworkplan")
       ?: throw MissingQueueException("HmppsQueue educationandworkplan not found")
   }
-  val domainEventQueueDlqClient by lazy { domainEventQueue.sqsDlqClient }
   val domainEventQueueClient by lazy { domainEventQueue.sqsClient }
+  val domainEventQueueDlqClient by lazy { domainEventQueue.sqsDlqClient }
+
+  val inductionScheduleEventQueue by lazy {
+    hmppsQueueService.findByQueueId("inductionscheduleeventqueue")
+      ?: throw MissingQueueException("HmppsQueue testeventqueue not found")
+  }
+  val testInductionScheduleEventQueueClient by lazy { inductionScheduleEventQueue.sqsClient }
```

Comment:
in order to test that the outbound message was put on the topic we need to create a queue that we can then read the message back from - this is her.